Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -94,7 +97,7 @@ class TIFFLoadingContext {
                 Optional<Color> last_color {};
 
                 for (u32 column = 0; column < *m_metadata.image_width(); ++column) {
-                    auto color = Color { TRY(byte_reader()), TRY(byte_reader()), TRY(byte_reader()) };
+                    auto color = Color { TRY(bit_stream->template read_value<u8>()), TRY(bit_stream->template read_value<u8>()), TRY(bit_stream->template read_value<u8>()) };
```

Comment:
what's the point of using a bit_stream here? can't you just index into `decoded_bytes` now?

Reply:
That's me anticipating support for custom bit-depth, I shouldn't have put it in this commit.

Reply:
Actually, the stream allows us to return an `Error` instead of panicking if the strip does not contain enough data. So I kept it, but I removed the `BitStream` which was truly useless in this commit.